homeassistant/components/lg_netcast/const.py

```diff
@@ -1,3 +1,15 @@
 """Constants for the lg_netcast component."""
 
+from datetime import timedelta
+from typing import Final
+
+ATTR_MANUFACTURER: Final = "LG"
+ATTR_MODEL_NAME = "model_name"
+ATTR_FRIENDLY_NAME = "friendly_name"
+ATTR_UUID = "uuid"
```

Comment:
Please remove these constants, they're only used once, in config_flow.py

Reply:
Done